bluemira/magnetostatics/trapezoidal_prism.py

```diff
@@ -367,46 +371,10 @@ def __init__(
         self.alpha = alpha
         self.beta = beta
         # Current density
-        self.rho = current / (4 * breadth * depth)
+        self.area = 4 * breadth * depth
```

Comment:
remove the breadth and depth lines above too if you take my earlier suggestion
```suggestion
        self._breadth_depth_area(breadth, depth)
```